client/src/components/Ticketing/ticketingmanager/Event/components/EventShowingView.tsx

```diff
@@ -24,6 +32,35 @@ export const EventShowingView = (props: EventInstanceViewProps) => {
       .slice(0, 8)}`,
   );
 
+  const onCloneSuccess = async (res) => {
+    const data = await res.json();
+    setReloadShowing((reload) => !reload);
+    setPopUpProps(
+      'Success',
+      'Showing successfully cloned',
+      true,
+      `update-modal-showing-id-${data.eventinstanceid}`,
```

Comment:
Could you rename this to `clone-modal...`

Reply:
Fixed